CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+- [NEW] Configuration option `createDb` for creating the database. Can be disabled to allow use of a
```

Comment:
I think this should say `cloudant.createDb` to avoid confusion - are the un-prefixed properties user-visible anywhere?

Reply:
Fixed in 226121e

Reply:
Copyrights fixed in 422ed8e